packages/create/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "@sap-ux/create",
+    "description": "SAP Fiori tools module to add or remove features",
```

Comment:
Why call the module `create` when it can add or remove features? `modify-feature` or something works better?

Reply:
Thanks for reminding me about this. It is handy to be able to do that with `npm init`.

My concern was with running with `npx` or `npm exec`. Then the command will look something like:
`npx @sap-ux/create remove <feature>`

Reply:
We still have a `bin` entry in `package.json`:
```json
"bin": {
    "create-fiori": "./dist/index.js"
},
```
https://github.com/SAP/open-ux-tools/pull/806/files#diff-84505d75821385613d447e5d9c00802c2b657d2fbed9b9ccc2303a1e751dcc3bR14-R15

If you have `@sap-ux/create` globally installed on your machine you can do something like
```
npx create-fiori add mockserver-config 
```
or, to follow your concern
```
npx create-fiori remove mockserver-config
```

I just named it `create-fiori`, but I think we could name it differently. Any proposals? I guess best would be `@sap-ux` to be consistent, but I'm not sure if a terminal command starting with `@` is the best idea. Opinions?

Reply:
Just did a test, `@sap-ux` can't be used for `bin`, but `sap-ux` would work.